backend/src/routes/doctorOrders.js

```diff
@@ -359,7 +358,7 @@ async function parseNCPDPScript(newRx) {
     dispenseStatus: 'Pending'
   };
 
-  const isRemsDrug = !!getRemsAdminFhirUrl(incompleteOrder);
+  const isRemsDrug = !!getRemsAdminEtasuUrl(incompleteOrder);
```

Comment:
How would this work if there was an intermediary setup and not every drug with a REMS listed in the configuration? Do we need to find a way to query the intermediary to check if it is a REMS drug?

Reply:
That's a good question- I suppose the pharmacy wouldn't know innately if Tylenol/ibuprofen was a REMS drug. Maybe if the intermediary had an endpoint that returns a boolean based on what the DoctorOrder is (drug id, is using the rxnorm system url), if the drug isn't a REMS drug then we use whatever default FHIR url the pharmacy has coded in.

Reply:
is your question/feature suggestion is related to https://jira.mitre.org/browse/REMS-709?